extensions/vscode/src/extension.ts

```diff
@@ -145,11 +144,13 @@ async function startClient(context: ExtensionContext) {
   console.log("Starting client")
 
   const ftlConfig = vscode.workspace.getConfiguration("ftl")
-  const ftlPath = ftlConfig.get<string>("executablePath") ?? "ftl"
+
   const workspaceRootPath = await getProjectOrWorkspaceRoot()
-  const ftlAbsolutePath = path.isAbsolute(ftlPath) ? ftlPath : path.resolve(workspaceRootPath, ftlPath)
+  const resolvedFtlPath = resolveFtlPath(workspaceRootPath, ftlConfig)
+
+  console.log('ftl path', resolvedFtlPath)
```

Comment:
did you want to remove this, or do we actually use console logs in prod?

Reply:
Yeah it's for debugging, but it actually only really shows in the debug log so I'll keep it for now! Thanks for catching it though! :)